### PR TITLE
[breakpad] Add windows tools

### PR DESCRIPTION
**Pull request by RealChuan**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [ ] ~~SHA512s are updated for each updated download.~~
- [x] The "supports" clause reflects platforms that may be fixed by this new version.
- [ ] ~~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~~
- [ ] ~~Any patches that are no longer applied are deleted from the port's directory.~~
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by jimwang118** [comment]
Testing under local Windows did not generate dump_syms, sym_upload, google_converter.

**Comment by RealChuan** [comment]
> Testing under local Windows did not generate dump_syms, sym_upload, google_converter.

Use vcpkg install breakpad[tools] - triplet x64-windows?

**Comment by jimwang118** [comment]
> > Testing under local Windows did not generate dump_syms, sym_upload, google_converter.
> 
> Use vcpkg install breakpad[tools] - triplet x64-windows?

I'm very sorry, it's my fault that I didn't add the feature during the test. Retested and passed.

**Comment by jimwang118** [comment]
Compile test pass with following triplets:
```
x64-windows
```

File: ports/breakpad/CMakeLists.txt [hunk 1]
```diff
@@ -178,6 +178,29 @@ if(INSTALL_TOOLS)
             src/tools/linux/core_handler/core_handler.cc)
         target_link_libraries(core_handler PRIVATE libbreakpad_client)
         install(TARGETS core_handler DESTINATION bin)
+    elseif(WIN32)
+        set(CMAKE_GENERATOR_PLATFORM ${VCPKG_TARGET_ARCHITECTURE})
```
Comment (dg0yt) [inline ports/breakpad/CMakeLists.txt:182]: Isn't it initialized correctly?  
"The value of this variable should never be modified by project code."  
https://cmake.org/cmake/help/latest/variable/CMAKE_GENERATOR_PLATFORM.html

**Comment by RealChuan** [comment]
Test https://github.com/RealChuan/vcpkg/actions/runs/9078244060

File: ports/breakpad/FindDiaSDK.cmake [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+# i.e. C:/Program Files (x86)/Microsoft Visual Studio/2019/Community
+set(VS_PATH ${CMAKE_GENERATOR_INSTANCE})
+get_filename_component(VS_DIA_INC_PATH "${VS_PATH}/DIA SDK/include" ABSOLUTE CACHE)
+
+# Starting in VS 15.2, vswhere is included.
+# Unclear what the right component to search for is, might be Microsoft.VisualStudio.Component.VC.DiagnosticTools
+# (although the friendly name of that is C++ profiling tools).  The toolset is the most likely target.
+set(PROGRAMFILES_X86 "ProgramFiles(x86)")
+execute_process(
+  COMMAND "$ENV{${PROGRAMFILES_X86}}/Microsoft Visual Studio/Installer/vswhere.exe" -latest -products * -requires Microsoft.VisualStudio.Component.VC.Tools.x86.x64 -property installationPath
```
Comment (data-queue) [inline ports/breakpad/FindDiaSDK.cmake:15]: @BillyONeal pointed out that this command may choose a DIA SDK instance not related to the compiler used

**Review by data-queue — CHANGES_REQUESTED** [review]

**Pull request closed without merging**